packages/katello/katello/katello.spec

```diff
@@ -58,8 +58,6 @@ Requires: mod_xsendfile
 Requires(post): candlepin >= 2.0
 Requires: rubygem-foreman_maintain >= 0.2.2
 Requires: candlepin-selinux >= 2.0
-Requires: java-openjdk >= 1:1.7.0
-Requires: java-openjdk < 1:1.8.0.45
 Requires: /usr/bin/psql
 Requires: /usr/bin/pg_dump
```

Comment:
These can probably be removed as well because foreman_maintain now has the backup scripts that require these

Reply:
I can do a targeted further update to clean this up but I'd like to keep this change focused.